freeipa.spec.in

```diff
@@ -1188,6 +1188,18 @@ if [ -f '/etc/ssh/sshd_config' -a $restore -ge 2 ]; then
 
         /bin/systemctl condrestart sshd.service 2>&1 || :
     fi
+    # If the snippet has been created, ensure that it is included
+    # either by /etc/ssh/sshd_config.d/*.conf or directly
+    if [ -f '/etc/ssh/sshd_config.d/04-ipa.conf' ]; then
+        if ! grep -E -q  '^\s*Include\s*/etc/ssh/sshd_config.d/\*\.conf' /etc/ssh/sshd_config 2> /dev/null ; then
```

Comment:
Should the grep include -i to ignore case, just in case? The sshd_config(5) keywords are case-insensitive.

Reply:
That becomes tricky... the keyword `Include` is case-insensitive but the value isn't. The regexp would then be `[In][Nn][Cc][Ll][Uu][Dd][Ee]`. Note that we didn't consider up till now that keywords could be case-insensitive in this file, meaning that if we start changing it we should be consistent and also do it for the other keywords such as AuthorizedKeysCommand, PubKeyAgent etc... that are also grep'ed.

Reply:
For the 80/20 rule I think case sensitive is fine, leave it as-is.